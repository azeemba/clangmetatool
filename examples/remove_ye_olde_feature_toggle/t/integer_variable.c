#include <ye_olde_feature_toggle.h>
int test_function() {
  int feature_id = 3;
  if (ye_olde_feature_toggle_is_enabled(feature_id))
    return 1;
  feature_id = 5;
  if (ye_olde_feature_toggle_is_enabled(feature_id))
    return 0;
  return 42;
}
