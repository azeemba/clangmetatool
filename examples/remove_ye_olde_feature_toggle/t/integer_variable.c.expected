int test_function() {
  int feature_id = 3;
  if (1 /* removed ye_olde_feature_toggle 3 */)
    return 1;
  feature_id = 5;
  if (0 /* removed ye_olde_feature_toggle 5 */)
    return 0;
  return 42;
}
